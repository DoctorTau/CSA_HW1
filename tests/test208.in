250
-4511 -3138 -584 2454 -5368 4465 652 -5619 -3833 5909 798 -8026 -2167 1581 -2982 -4815 731 1782 3570 8838 -5191 -3374 -8931 -652 765 -3581 -2123 8745 2146 -4103 5276 4311 1841 4193 7235 8247 -5326 -4731 -1258 2812 -7279 3834 -6490 -394 2243 7013 -4650 -1758 2280 -8363 3407 -2982 8771 -6051 2251 -7245 4420 2793 -8113 1910 43 9254 2455 -1904 6034 -7670 8135 -1530 3455 -369 -2979 1782 4083 -789 6199 -3123 -284 -1149 8045 5980 -9117 -6127 -8548 1645 -4650 -6718 7101 7385 -1995 -1424 -4140 5353 6992 9109 -9563 1556 7295 8119 -7080 -1990 571 -8809 479 8226 -3238 8007 3581 -7576 -7202 -7951 -9069 -5427 -2078 575 3474 2850 -2999 -2079 9204 -3355 -8251 1087 -1852 3788 -6468 3345 -2203 -8518 1679 3401 -5496 -4171 -1745 -1491 5171 5856 9453 4251 7827 -2372 9749 -9854 -8953 9957 -9546 -9087 -5219 3285 -1011 -8848 5161 22 6248 8466 -6808 9856 1581 8948 -7333 9853 -2666 1975 683 -5819 4056 -2477 3318 4538 -459 2379 -3249 -1427 -1129 -260 -6031 2496 2472 5217 137 -1515 5827 -1602 -2277 -2142 -400 3695 -9018 3642 -1353 -5891 -2983 7822 4082 5152 -4178 -8929 632 5554 7975 -6928 -307 7908 -8833 9868 -7023 9015 3065 689 8368 7571 8475 -3328 -8666 -6752 8800 6828 2781 5119 -6393 -5483 8182 2484 8237 -2753 5679 3915 -6635 6270 9687 -8252 -5091 1572 1317 -6873 2839 -7214 -9689 6139 7860 -7769 -6448 -3390 8722 5454 -3202 676 9300 7299 -9521 -9363 