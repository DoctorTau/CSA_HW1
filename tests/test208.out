-3138 -584 -5368 -8026 -2982 -3374 -652 -5326 -1258 -6490 -394 -4650 -1758 -2982 -1904 -7670 -1530 -284 -8548 -4650 -6718 -1424 -4140 -7080 -1990 -3238 -7576 -7202 -2078 -1852 -6468 -8518 -5496 -2372 -9854 -9546 -8848 -6808 -2666 -260 -1602 -2142 -400 -9018 -4178 -6928 -3328 -8666 -6752 -8252 -7214 -6448 -3390 -3202 